### ISSUE TITLE
Add additional score info to Braille panel

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#20155]
### Your idea

Currently the Braille panel includes, notes, octave marks, fingering, dynamics, articulation & ornaments, as well as various other marks like bow markings, some repeats.
The most notable omission is pedal marks, expression text and stave text.
But it would be nice as time goes on to include everything that is in the print such as lyrics, chord symbols, roman numerals, figured bass, etc.

### Problem to be solved

Having all the score info included on a Braille display would mean a blind user does not have to use a screen reader to hear the score read out and can get to all the information much more easily.

**Comment by jmwmusic** [comment]
A number of users have requested String signs be added to the Braille output.  Maybe these could be given some priority in any list of symbols to add.
